core/tests/test_log_validation.py

```diff
@@ -1,6 +1,6 @@
 import unittest
 
-from core.util.audit_logs_parsing import Log
+from core.replay.connections_parser import Log
```

Comment:
Fix for using connectionLog from `connection_parser.py` and removed `audit_logs_parsing`